src/ImageSharp/Formats/Jpeg/JpegDecoderCore.cs

```diff
@@ -73,6 +74,16 @@ internal sealed class JpegDecoderCore : IRawJpegData, IImageDecoderInternals
         /// </summary>
         private byte[] iptcData;
 
+        /// <summary>
+        /// Whether the image has a XMP data.
+        /// </summary>
+        private bool isXmp;
```

Comment:
Naming: `hasXmp`

Reply:
There are already 3 other properties with this naming convention, `isExif`, `isIptc` and `isIcc`. 
Shall I update all 4 then ?

Reply:
Yes please!

Reply:
With pleasure !